cmd/openshift-install/main.go

```diff
@@ -104,6 +104,7 @@ func runRootCmd(cmd *cobra.Command, args []string) {
 		ForceColors:            terminal.IsTerminal(int(os.Stderr.Fd())),
 		DisableTimestamp:       true,
 		DisableLevelTruncation: true,
+		DisableQuote:           true,
```

Comment:
this eliminates the escaping logic they've been providing?  I like the change, but I recognize that this could change machine parsability in a way to splitting newlines does not, right?

Reply:
For users with interactive terminal this has no effect.
For non-interactive outputs like CI pods, this will no quote the message and yes I think this will make the message show up on multiple lines instead of one line.
but the other change will already have split up the message on newline so this only makes the message look cleaner with no unnecessary escaping.